pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -206,7 +206,14 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 		obj.KubeAPIBurst = 10
 	}
 	if obj.SerializeImagePulls == nil {
-		obj.SerializeImagePulls = utilpointer.BoolPtr(true)
+		// SerializeImagePulls is default to true when MaxParallelImagePulls
+		// is not set, and false when MaxParallelImagePulls is set.
+		// This is to save users from having to set both configs.
+		if obj.MaxParallelImagePulls == nil {
```

Comment:
if we allow MaxParallelImagePulls to be 1 now, we need this:

```suggestion
		if obj.MaxParallelImagePulls == nil || *obj.MaxParallelImagePulls < 2 {
```

Reply:
Good catch. Updated the code. Thanks!